redisinsight/ui/src/slices/instances/sentinel.ts

```diff
@@ -164,7 +164,13 @@ export function fetchMastersSentinelAction(
 
 // Asynchronous thunk action
 export function createMastersSentinelAction(
-  payload: CreateSentinelDatabasesDto,
+  payload: {
```

Comment:
Note: To be honest, I'm surprised that the types differ so much. Either the original implementation was wrong and didn't care about the types at all, or we have a huge problem with our types being out of sync between BE/FE.